src/govuk/components/character-count/character-count.unit.test.mjs

```diff
@@ -101,12 +91,15 @@ describe('CharacterCount', () => {
 
             const component = new CharacterCount($div, {
               i18n: {
-                charactersUnderLimitOne: 'Different custom text. Count: %{count}'
+                charactersUnderLimit: {
```

Comment:
Using `tsc` is brilliant at spotting things like this

We removed **charactersUnderLimitOne** from the JSDoc and it spotted all the places we no longer matched the type declaration